topo/bond/faucet/faucet.yaml

```diff
@@ -232,3 +235,6 @@ vlans:
   171:
     description: Faucet IoT
     edge_learn_stack_root: true
+  272:
+    description: Orchestrated Testing
+    edge_learn_stack_root: true
```

Comment:
edge_learn_stack_root should no longer be necessary so leave it out (ideally remove the other instance too while we're at it)

Reply:
Done.